doc/ForcedUpdate.md

```diff
@@ -0,0 +1,30 @@
+# Forced Update
+
+The forced update feature is for when you want to force the user to update the app.
+You could use this, for example, when the backend changes and you do not want the users to still use the old API.
+
+To force an update, we suscribe to the `UpdateRequired` Event from the `UpdateRequiredservice` in the `CoreStartup`.
+
+```cs
+ private void SuscribeToRequiredUpdate(IServiceProvider services)
```

Comment:
Do we need to copy the code into the documentation? I think it might be better to just link to it instead. That way, if it changes, we don't need to duplicate all the changes in the documentation.